testController.js

```diff
@@ -1,8 +1,37 @@
 document.addEventListener('DOMContentLoaded', function() {
-    document.getElementById('home').addEventListener('click', function() {
-        console.log('vleze')
-        getWeather('sofia').then(function(data) {
-            console.log(data)
+        function getCity() {
+            return document.getElementById('city').value
+        }
+    document.getElementById('enterCity').addEventListener('click', function() {
+        let city = document.getElementById('city').value 
+        getWeather(getCity()).then(function(data) {
+            let weatherToday = data;
+            document.getElementById('infoWraper').style.display = 'block';
+            document.getElementById('cityName').textContent = `Град: ${weatherToday.name}`;
```

Comment:
Here you can use the es6 string templates with caution because the business requirements can demand that your app works on older versions of browsers that doesn't implement this es6 feature. So in that case you should use some sort of transpiler (babel for example) to transform the es6 code to es5.